docs/data-sources/dcs_product.md

```diff
@@ -6,6 +6,9 @@ subcategory: "Distributed Cache Service"
 
```

Comment:
update the subcategory to `Deprecated` as well

Reply:
done